api-catalog-ui/frontend/src/components/ServiceTab/ServiceTab.jsx

```diff
@@ -24,9 +24,6 @@ export default class ServiceTab extends Component {
             selectedVersion: null,
```

Comment:
Please check line 10, `IconButton` is no longer used and will make the linter fail

Reply:
ok, I will fixed it